02-fundamentals.Rmd

```diff
@@ -480,6 +480,14 @@ Here's the process on how to get it done.
 3 | Product Design & Development | Wireframing, prototyping, and application development are the main engines to transform an idea into a final product. Products can range from static presentations/reports to full interactive applications. Mobile, tablet, TV, and workstation platforms can all be mediums to help deliver the final product. The secret to a great end product is how well the data story is conceptualized. If the story is weak then the end product will also suffer. 
 4 | QA & Product Release | The best part of any project is to get it finalized and released for all to see. All data gets verified for accuracy, functionality testing (if applicable), application flow (if applicable), design testing, and remaining items are all completed. The end result is an engaging visual product for all intended audiences to see and use.
 
+### Demand for Data Literacy 
+
+The demand for data literacy is at an all-time high. Originally, data science was focused on the finance and tech industries but the demand for data science skills is increasing for every industrial section. ([Lifeline)](https://lifelinedatacenters.com/data-center/business-intelligence/))
```

Comment:
I think you got the brackets here mixed up a little. it should be `([Lifeline](https://lifelinedatacenters.com/data-center/business-intelligence/))`

Reply:
Good Catch. Thanks!